net/base/load_timing_info.h

```diff
@@ -203,6 +203,7 @@ struct NET_EXPORT LoadTimingInfo {
   uint64_t encoded_body_size;
   std::string alpn_negotiated_protocol;
   std::string connection_info_string;
+  base::TimeTicks response_end;
```

Comment:
There's a number of `basse::TimeTicks` above already - doesn't one of those correspond with what we need ? I think in last version of `//net` we used we didn't have it or didn't know how to plumb through.